library/src/main/java/com/novoda/downloadmanager/DownloadManagerBuilder.java

```diff
@@ -12,6 +19,8 @@
  */
 public class DownloadManagerBuilder {
 
+    private static final int ANDROID_N = 24;
```

Comment:
Maybe we should bump the target SDK version given we code against it and the changes are specifically to support N+